client/frontend/package.json

```diff
@@ -17,6 +17,8 @@
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.1",
+    "react-scroll-to": "^3.0.0-beta.6",
+    "react-transition-group": "^4.3.0",
     "redux": "^4.0.5"
```

Comment:
Remove unused imports "react-scroll-to": "^3.0.0-beta.6", "react-transition-group": "^4.3.0", "devDependencies": {
    "node-sass": "^4.13.1"
  }